it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/LoadTestBase.java

```diff
@@ -495,6 +495,17 @@ public static PipelineOperator.Config createConfig(LaunchInfo info, Duration tim
         .build();
   }
 
+  public static PipelineOperator.Config createConfig(
```

Comment:
Please add an ignore block in codecov for these files.

Reply:
We had added this to codecov just as an FYI. These don't block the PR. Only coverage drop in Spanner templates is a required check.